cmd/netgoal/generate.go

```diff
@@ -465,19 +466,29 @@ func saveGoalTemplateToDisk(template netdeploy.NetworkTemplate, filename string)
 }
 
 func generateWalletGenesisData(wallets, npnHosts int) gen.GenesisData {
+	ratZero := big.NewRat(int64(0), int64(1))
+	ratHundred := big.NewRat(int64(100), int64(1))
 	data := gen.DefaultGenesis
+	totalWallets := wallets + npnHosts
+	data.Wallets = make([]gen.WalletData, totalWallets)
+	participatingNodeStake := big.NewRat(int64(100), int64(wallets))
+	nonParticipatingNodeStake := ratZero
 	if npnHosts > 0 {
-		wallets = 2 * wallets
+		// split participating an non participating stake evenly
```

Comment:
Have we always been assuming an even split between participating and non participating nodes? I wonder if that should be configurable. It looks like that would be a new feature though so probably out of scope